### DISCUSSION TITLE
[Feature] Filter search by owner

### DISCUSSION THREAD

**Discussion by yurividal** [immich-app/immich#13348] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Since allowing users to delete/favorite a partner-shared photo is not implemented and doesn't seem to be in the roadmaps, it would be useful if there was a filter on the search to only show items owned by me. 
Currently, whenever i want to search for specific category of photos, to bulk delete, immich shows photos owned by me and by my partner. As a result, i cannot delete nor favorite in bulk, since it gives me an error. So, i have to manually go to settings, disable "Partner Sharing -> Show in timeline", perform the action, then re-enable Show in timeline. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by coltenkrauter** [comment]
This would be nice.

**Comment by leggt** [comment]
May I add to this suggesting by allowing to filter on any owner? Being able to see which photos my wife or kids own/uploaded would be cool.

Edit: I see now that you can see partner's photos in the sharing tab. But it would still be nice to add to the search filters

**Comment by soupyfish** [comment]
being able to filter by owner in a shared album would be very useful, at the moment the only way to tell who uploaded an asset is to check the asset metadata
